### DISCUSSION TITLE
:warning: Git security vulnerabilities announced - upgrade to the latest version

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#44687] [Announcements]
Today, the Git project [released](https://lore.kernel.org/git/xmqq7cxl9h0i.fsf@gitster.g/T/#u) new versions to address a pair of security vulnerabilities, [CVE-2022-41903](https://github.com/git/git/security/advisories/GHSA-475x-2q3q-hvwq), and [CVE-2022-23521](https://github.com/git/git/security/advisories/GHSA-c738-c5qq-xg89), that affect versions 2.39 and older.

**Git users are encouraged to upgrade to the latest version, especially if they use `git archive`, work in untrusted repositories, or use Git GUI on Windows.**

For more details, please visit our [blog post](https://github.blog/2023-01-17-git-security-vulnerabilities-announced-2/).

**Comment by 681Sad** [comment]
hola, soy el dueño de ese git pero no puedo recuoerarlo, el que lo edito esta cometiendo un crimern , todos ven mis datos personales

**Comment by sotmodan** [comment]
ayuda para revisar una apllicacion que me esta causando errores en mi celular y de mas aparatos

**Reply by GF-Elite** [reply]
gh

**Comment by ghost** [comment]


**Reply by bestcox** [reply]
Yes

**Comment by joel8485** [comment]
https://github.com/orgs/community/discussions/44687#discussioncomment-473440

**Comment by joel8485** [comment]
#40454

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
